### ISSUE TITLE
SpectralPowerPipeline2D cannot be unpickled

### ISSUE THREAD

**Issue by seidlj** [raysect/source#411]
Hello,

when I try to unpickle `raysect.optical.observer.SpectralPowerPipeline2D` object, it raises 
```
File ~/.local/lib/python3.10/site-packages/raysect/optical/observer/pipeline/spectral/power.pyx:390, 
in raysect.optical.observer.pipeline.spectral.power.SpectralPowerPipeline2D.__setstate__()

TypeError: Expected tuple, got int
```

I've discussed that with @Mateasek, who thinks that this is caused by incompatibility of the `__getstate__` and `__setstate__` methods of that class. The following code raises the same error:

```python
from raysect.optical.observer import SpectralPowerPipeline2D

pipeline = SpectralPowerPipeline2D()
pipeline.__setstate__(pipeline.__getstate__())
```

Raysect version: 0.8.0

**Comment by Mateasek** [comment]
Hi @seidlj , the problem is wrong typing [here](https://github.com/raysect/source/blob/3c1f2f2f1cce9c2a2d5ca2952a70af3287d53dbf/raysect/optical/observer/pipeline/spectral/power.pyx#L390):
`self._pixels = 0`

I think the fix could be changing it to :
`self._pixels = None`

**Comment by CnlPepper** [comment]
Fixed in development branch (v0.8.1).
